apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/RetryOptions.cs

```diff
@@ -56,13 +56,13 @@ public enum ExceptionHandling
     public sealed class RetryOptions
     {
         /// <summary>The default buffer size in bytes. 2^18 = 262144.</summary>
-        private const int DefaultBufferSize = 262144;
+        public static readonly int DefaultBufferSize = 262144;
```

Comment:
Possibly don't document the current default value, or explicitly warn that it might change?